doc/datamodel_rstgen.py

```diff
@@ -0,0 +1,10 @@
+"""Provides a module for generating the Fluent datamodel result files."""
+
+from ansys.fluent.core import FluentMode, launch_fluent
+from ansys.fluent.core.utils.fluent_version import get_version_for_file_name
+from codegen.datamodelgen import generate
+
+if __name__ == "__main__":
+    sessions = {FluentMode.SOLVER: launch_fluent()}
+    version = get_version_for_file_name(session=sessions[FluentMode.SOLVER])
+    generate(version, None, sessions, True, False)
```

Comment:
Will this launch Fluent? We have to do the docgen without launching Fluent. I think the generated API classes have all information to write out the .rst files.

Reply:
We are going to remove this code. We will discuss this soon.